content/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -0,0 +1,307 @@
+
+## Memeriksa adaptor jaringan 
+
+Jika kamu memiliki lebih dari satu adaptor jaringan, dan komponen Kubernetes tidak dapat dijangkau melalui rute bawaan (_default route_),
+kami merekomendasikan kamu untuk menambahkan rute IP sehingga alamat-alamat klaster Kubernetes melewati adaptor yang tepat.
+
+## Membuat iptables melihat _bridged traffic_
+
+Agar iptables pada Node Linux dapat melihat _bridged traffic_ dengan benar, kamu harus memastikan `net.bridge.bridge-nf-call-iptables` bernilai 1 pada pengaturan `sysctl`, misalnya.
+
+```bash
+cat <<EOF | sudo tee /etc/sysctl.d/k8s.conf
+net.bridge.bridge-nf-call-ip6tables = 1
+net.bridge.bridge-nf-call-iptables = 1
+EOF
+sudo sysctl --system
+```
+
+Pastikan modul `br_netfilter` sudah dimuat sebelum melakukan langkah ini. Hal ini dilakukan dengan menjalankan `lsmod | grep br_netfilter`. Untuk memuatnya secara eksplisit gunakan `sudo modprobe br_netfilter`.
+
+Untuk detail lebih lanjut, silakan lihat laman [Network Plugin Requirements](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements).
+
+## Memeriksa port yang dibutuhkan
+
+### Node _control-plane_
+
+| Protocol | Direction | Port Range | Purpose                 | Used By                   |
+|----------|-----------|------------|-------------------------|---------------------------|
+| TCP      | Inbound   | 6443*      | Kubernetes API server   | All                       |
+| TCP      | Inbound   | 2379-2380  | etcd server client API  | kube-apiserver, etcd      |
+| TCP      | Inbound   | 10250      | Kubelet API             | Self, Control plane       |
+| TCP      | Inbound   | 10251      | kube-scheduler          | Self                      |
+| TCP      | Inbound   | 10252      | kube-controller-manager | Self                      |
+
+### Node pekerja (_worker_)
+
+| Protocol | Direction | Port Range  | Purpose               | Used By                 |
+|----------|-----------|-------------|-----------------------|-------------------------|
+| TCP      | Inbound   | 10250       | Kubelet API           | Self, Control plane     |
+| TCP      | Inbound   | 30000-32767 | NodePort Services†    | All                     |
+
+† Jangkauan port bawaan untuk [Service NodePort](/docs/concepts/services-networking/service/).
+
+Angka port yang ditandai dengan * dapat diganti (_overrideable_), sehingga kamu harus memastikan port _custom_ lainnya yang kamu sediakan juga terbuka.
+
+Meskipun port etcd turut dituliskan pada Node control-plane, kamu juga bisa menghos klaster etcd-mu sendiri
+secara eksternal atau pada port _custom_.
+
+_Plugin_ jaringan Pod yang kamu gunakan (lihat di bawah) juga mungkin membutuhkan port tertentu untuk terbuka.
+Karena hal ini dapat berbeda pada setiap _plugin_ jaringan Pod, silakan lihat
+dokumentasi _plugin_ mengenai port yang dibutuhkan.
+
+## Menginstal _runtime_ 
+
+Untuk menjalankan kontainer pada Pod, Kubernetes menggunakan
```

Comment:
Container is kubernetes API object
```suggestion
Untuk menjalankan Container pada Pod, Kubernetes menggunakan
```

Reply:
Ah, I didn't realize that container is also an API object... Fixed in 0b74a869bea33a4eb86e43a89b5284bcd244444f